handlers.go

```diff
@@ -402,6 +402,10 @@ func profileAddHandler(w *Web) {
 	if port := getEnv("SUBSPACE_LISTENPORT", "nil"); port != "nil" {
 		listenport = port
 	}
+	allowedips := "0.0.0.0/0, ::/0"
+	if ips := getEnv("ALLOWED_IPS", "nil"); ips != "nil" {
```

Comment:
This environment variable should probably be prefixed with `SUBSPACE_` to keep it in line with the others